Core/HLE/sceNp2.cpp

```diff
@@ -0,0 +1,566 @@
+// Copyright (c) 2012- PPSSPP Project.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 2.0 or later versions.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License 2.0 for more details.
+
+// A copy of the GPL 2.0 should have been included with the program.
+// If not, see http://www.gnu.org/licenses/
+
+// Official git repository and contact information can be found at
+// https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
+
+// This is pretty much a stub implementation. Doesn't actually do anything, just tries to return values
+// to keep games happy anyway.
+
+#include <StringUtils.h>
+#include "Core/MemMapHelpers.h"
+#include <Core/CoreTiming.h>
```

Comment:
Should use `""`.  Neither of these are system headers from the OS or installed libraries, we want our include path to take precedence.

-[Unknown]